### DISCUSSION TITLE
The storage time of the temporary object has expired

### DISCUSSION THREAD

**Discussion by horyachka** [totumonline/totum-ru-issues-and-discussions#572] [ОШБК!] [answered]
Добрый день,
В процессе создания нового поля таблицы циклов (после нажатия стандартной кнопки Add field) и его настройки проходит от 20 до 40 секунд и появляется окно с текстом: ![totum_error](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/91678200/d965df1c-fd33-4a79-aec7-1c416c16f43a). Далее нужно это поле закрыть без возможности сохранения и начать все заново.
Такая ситуация сложилась после переноса проекта из VDS сервера на выделенный сервер. 
Подскажите пожалуйста: что нужно сделать, чтобы эта надпись не появлялась? Готовность проекта – 90% и пользователи еще не работают.
Параметры сервера: AMD Ryzen 9 5950X / 128 GB ECC RAM / 2x SSD U.2 NVMe 3,84 TB
Totum V 4.11.57.0
Спасибо!

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Мы видели эту ошибку. Крон, который удаляет tmp запускается раз в 10 минут и для нового поля там какая-то ошибка с датой. Те если начать создавать поле в 9:30 то следующий крон его сотрет через 30 сек.

В V5 будет исправлено. Что можно сделать сейчас: создаете поле, задаете ему тип и название и сохраняете. Потом открываете на редактирование и работаете по его настройке. Проблема существует только с полем, которое еще не записано в БД.

**Comment by horyachka** [comment]
Добрый день, эта ошибка появляется еще и при создании строк пользователями, которые вносят данные в таблицы.
Алексей, напишите, пожалуйста, путь к этому крону. Попробуем изменить эти 10 минут на другое время. 
Спасибо!

**Reply by ttm-alexeypolunin** [reply]
От пользователя totum
```bash
su totum
```

Открываете crontab
```
crontab -e
```
Крон clean-schemas-tmp-tables
```
*/10 * * * * cd /home/totum/totum-mit/ && bin/totum clean-schemas-tmp-tables
```

меняете `*/10` —> `*/30` будет раз в 30 минут

или `*/10 * * * *` —> `0 3 * * *` один раз в день в 3 часа ночи

`минуты` `часы` `день_месяца` `месяц` `день_недели`

**Reply by ttm-alexeypolunin** [reply]
Исправлено V5
